.github/workflows/release.yaml

```diff
@@ -42,8 +42,23 @@ jobs:
       - name: Run tests
         run: tox -e unit
 
-  integration-test:
-    name: Integration tests
+  # TODO: Uncomment when the following bug is resolved
```

Comment:
if you add the following to your ci.yaml and release.yaml files, you can reuse your CI pipeline in this pipeline, reducing duplication: 

```yaml
# ci.yaml
on:
  workflow_call:
```
```yaml
# release.yaml
jobs:
  integration-test:
    uses: ./.github/workflows/ci.yaml
```

Reply:
ooh, this is a great suggestion! however, i'll follow up on this suggestion in a separate PR